src/zui/ZUIEditTextInPlace/index.tsx

```diff
@@ -143,15 +176,39 @@ const ZUIEditTextinPlace: React.FunctionComponent<ZUIEditTextinPlaceProps> = ({
     }
   };
 
+  const valueText = () => {
+    if ((autoOnEdit && !text) || (editing && !text)) {
+      return '';
+    }
+
+    if (
+      (!autoOnEdit && currentType !== placeholder) ||
+      (!autoOnEdit && !text)
+    ) {
+      return currentType;
+    }
+    if (text) {
+      return text;
+    }
+  };
   return (
-    <Tooltip arrow disableHoverListener={editing} title={tooltipText()}>
-      <FormControl style={{ overflow: 'hidden' }} variant="standard">
+    <Tooltip
+      arrow
+      disableHoverListener={editing}
+      placement="top"
+      title={tooltipText()}
+    >
+      <FormControl
+        onClick={(e) => e.stopPropagation()}
+        style={{ overflow: 'hidden' }}
+        variant="standard"
+      >
         <span ref={spanRef} className={classes.span}>
           {text || placeholder}
         </span>
         <InputBase
           classes={{
-            input: classes.input,
+            input: showBorder ? classes.showBorder : classes.input,
```

Comment:
Instead of toggling between different classes, it's possible to use a property to `useStyles()` which allows you to send in parameters to the styling. Here's an example from `ZUIHeader`. 

Style definitions (see lines 24 and 32):
https://github.com/zetkin/app.zetkin.org/blob/e7c6b35082cf3b3d9069033e35453e0ea006f1f3/src/zui/ZUIHeader.tsx#L24-L36

Using the styles:
https://github.com/zetkin/app.zetkin.org/blob/e7c6b35082cf3b3d9069033e35453e0ea006f1f3/src/zui/ZUIHeader.tsx#L76